node/ltnodeclient.js

```diff
@@ -93,10 +93,12 @@ function sbRequire(require, resolve, path) {
   if(!getSB(path)) {
     if(path.match(/[\\\/]/) && fpath.extname(path) == ".js") {
       var sb = sandboxes[path.toLowerCase()] = context(path);
```

Comment:
@rbecheras I for one would love to see that happen. It would be great if you could take care of the [TernJS plugin](https://github.com/mortalapeman/LT-TernJS) as well. If I had to request a feature it would definitely be _jump to function declaration_.